msal/sku.py

```diff
@@ -2,5 +2,5 @@
 """
 
 # The __init__.py will import this. Not the other way around.
-__version__ = "1.33.0b1"
+__version__ = "1.34.0b1"
```

Comment:
Please also revert this change.

We will only bump version number when we cut a release, and decide the actual number at that time.

Besides, bumping version number in each feature PR brings complications in case we would end up reverting the feature PR.